src/test/java/uk/gov/pay/api/it/ResourcesFiltersLocalRateLimiterITest.java

```diff
@@ -0,0 +1,203 @@
+
+import static com.jayway.restassured.RestAssured.given;
+import static com.jayway.restassured.http.ContentType.JSON;
+import static io.dropwizard.testing.ConfigOverride.config;
+import static io.dropwizard.testing.ResourceHelpers.resourceFilePath;
+import static javax.ws.rs.core.HttpHeaders.AUTHORIZATION;
+import static org.hamcrest.core.IsCollectionContaining.hasItem;
+import static org.junit.Assert.assertThat;
+
+/**
+ * PaymentResourceITestBase not extended as RedisDockerRule should not apply for this.
+ */
+public class ResourcesFiltersLocalRateLimiterITest {
+
+    private static final String API_KEY = ApiKeyGenerator.apiKeyValueOf("TEST_BEARER_TOKEN", "qwer9yuhgf");
+    private static final String GATEWAY_ACCOUNT_ID = "GATEWAY_ACCOUNT_ID";
+    private static final String PAYMENTS_PATH = "/v1/payments/";
+
+    private static final int AMOUNT = 9999999;
+    private static final String CHARGE_ID = "ch_ab2341da231434l";
+    private static final String CHARGE_TOKEN_ID = "token_1234567asdf";
+    private static final PaymentState CREATED = new PaymentState("created", false, null, null);
+    private static final RefundSummary REFUND_SUMMARY = new RefundSummary("pending", 100L, 50L);
+    private static final String PAYMENT_PROVIDER = "Sandbox";
+    private static final String RETURN_URL = "https://somewhere.gov.uk/rainbow/1";
+    private static final String REFERENCE = "Some reference";
+    private static final String EMAIL = "alice.111@mail.fake";
+    private static final String DESCRIPTION = "Some description";
+    private static final ZonedDateTime TIMESTAMP = DateTimeUtils.toUTCZonedDateTime("2016-01-01T12:00:00Z").get();
+    private static final String CREATED_DATE = DateTimeUtils.toUTCDateString(TIMESTAMP);
+    private static final Map<String, String> PAYMENT_CREATED = new ChargeEventBuilder(CREATED, CREATED_DATE).build();
+    private static final List<Map<String, String>> EVENTS = Collections.singletonList(PAYMENT_CREATED);
+    private static final Address BILLING_ADDRESS = new Address("line1", "line2", "NR2 5 6EG", "city", "UK");
+    private static final CardDetails CARD_DETAILS = new CardDetails("1234", "Mr. Payment", "12/19", BILLING_ADDRESS, "Visa");
+
+    private static final String PAYLOAD = paymentPayload(AMOUNT, RETURN_URL, DESCRIPTION, REFERENCE);
+    private ExecutorService executor = Executors.newFixedThreadPool(2);
+
+    private ConnectorMockClient connectorMock;
+    private PublicAuthMockClient publicAuthMock;
+
+    @Rule
+    public MockServerRule connectorMockRule = new MockServerRule(this);
+
+    @Rule
+    public MockServerRule publicAuthMockRule = new MockServerRule(this);
+
+    @Rule
+    public DropwizardAppRule<PublicApiConfig> app = new DropwizardAppRule<>(
+            PublicApi.class
+            , resourceFilePath("config/test-config.yaml")
+            , config("connectorUrl", connectorBaseUrl())
+            , config("publicAuthUrl", publicAuthBaseUrl())
```

Comment:
it would be good to add `config("redis.endpoint", "http://bad.url")` to show we're really falling back to the local rate limiting

Reply:
Agree, added random url